atlp-rwanda/space-bn-frontend
# #175743396 chore(router): set up react router

#### What does this PR do?

- Setup react router

#### Description of Task to be completed? 

- setup react router

- Configure react router

#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch **`ch-setup-react-router-#175743396`**

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- In your default browser navigate to **`http://localhost:3000`**

> You should see a Rendered welcoming message with nav links
> You can click to one link and you will be redirect to the corresponding page

#### Any background context you want to provide?

- N/A

#### What are the relevant Pivotal Tracker stories?

- [#175743396](https://www.pivotaltracker.com/n/projects/2476677/stories/175743396)

#### Screenshots (if appropriate)
<img width="1440" alt="react-router-setp" src="https://user-images.githubusercontent.com/54619678/99349091-c41dce00-28a3-11eb-965e-a83f5d0c8972.png">


#### Questions:

- N/A
